### DISCUSSION TITLE
"You cannot fork this repository at this time"

### DISCUSSION THREAD

**Discussion by OneOfTheMicahs** [community/community#114482] [New to GitHub] [answered]
This account is fairly new—maybe a week or two old—and it isn't able to fork any repositories. I've tried public, private, and the octocat forking example repo, but all return the same error, "You cannot fork this repository at this time".

(By the way, flashes don't appear on the site anymore. I had to view the response preview in the network tab of devtools to see that error flash.)

Any advice on how to get forking to work?

**Comment by OneOfTheMicahs** [comment]
Commenting to push this to the top as I still am unable to fork any repo.

**Comment by OneOfTheMicahs** [comment, marked as answer]
For anyone seeing this, the reason I can't fork is that my account is flagged.

I had no indication of that I was flagged, and only found out when I went to [upgrade my account](https://github.com/settings/billing/summary) to be able to open a support ticket!

Still waiting on a response for Github to unflag my account...

**Comment by sonalkarmakar** [comment]
I can't fork a public repository that literally hosts a Java file for adding 2 integers only. I can't create a repo and connect to it using HTTP, only being allowed to use SSH. I also have a new account, created in September this year.

I've heard similar stories from others about automated flagging like this. Most notable one that I can remember is when someone got banned from login for being flagged as spam, because of making jokes in the Linux kernel mirror, which isn't an official repo and everyone makes a joke there.

Microsoft needs to fix their "AI" from causing more problems than helping.
